pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -146,7 +146,7 @@ PARSER_END(VfParser)
 	| <GE: ">=" >
 	| <LT: "<" >
 	| <GT: ">" >
-	| <EXCL: ("!"|"~") >
+	| <EXCL: ("!"|"~"|"NOT") >
```

Comment:
Per my conversation with @sergeygorbaty, he ported `~` from the JSP spec. VisualForce does not really support it, but there is no harm having it, I guess. Safe to remove it though.